tests/wpt/meta/fetch/metadata/report.https.sub.html.ini

```diff
@@ -1,10 +1,3 @@
 [report.https.sub.html]
```

Comment:
This test was the reason I stumbled upon the missing exclamation mark. Since it also tests the report-uri, but then its headers.